src/webviews/apps/plus/graph/GraphWrapper.tsx

```diff
@@ -1347,6 +1380,59 @@ export function GraphWrapper({
 									<SlOption value="current">Current Branch</SlOption>
 								</SlSelect>
 							</GlTooltip>
+							<div className={`shrink ${!Object.values(excludeRefsById ?? {}).length && 'hidden'}`}>
+								<GlPopover
+									className="popover"
+									placement="bottom-start"
+									trigger="click focus"
+									arrow={false}
+									distance={0}
+								>
+									<GlTooltip placement="top" slot="anchor">
+										<button type="button" id="test" className="action-button">
+											<CodeIcon icon={`eye`} />
```

Comment:
We should use the same "eye with slash" icon that's used in the in-component one.